simulation/sub8_gazebo/diagnostics/set_model_position_test.py

```diff
@@ -57,10 +57,9 @@ def run_mission(srv, sub, nh):
 def main():
     nh = yield txros.NodeHandle.from_argv('test')
     world_position_actual = yield nh.subscribe('/world_odom', Odometry)
-    yield run_mission(None, world_position_actual, nh)
-    #nh.advertise_service('/gazebo/job_runner/mission_start', RunJob, lambda srv: run_mission(srv, world_position_actual, nh))
+    yield run_mission(None, map_position_actual, nh)
```

Comment:
I think it was based on the old framework for setting up tests. All of the files in `gazebo_tests/` were set up with the new framework.
Trying to run anything from here would break due to the broken dependency with the old kill handling, you're right.